free-1.md

```diff
@@ -57,12 +73,16 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+This depends on their understanding of physics and how to do simulations. Most people will probably spend a lot of time implementing systems and may even not get around to adding many language aspects due to the long time to implement each.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+Pretty ambitious. It would be recommended to target a small domain within physics and do that well rather than trying to support everything.
```

Comment:
Agreed - this is also important since you cited feature bloat as a problem with existing tools.